src/components/Footer/components/SocialLink/SocialLink.jsx

```diff
@@ -0,0 +1,7 @@
+export default function SocialLink({ children, ...rest }) {
+    return (
+        <a {...rest}>
+            {children}
+        </a>
+    )
+}
```

Comment:
If we want to customize some links, but haven't done yet, you can leave a TODO comment with text about what you will doing, or why you left it.
Because in this component you created the main HTML tag and nothing else. So now I don't see any reason to use this.